apps/examples/lcd_test/example_lcd.c

```diff
@@ -254,13 +254,93 @@ static void test_bit_map(void)
 	free(lcd_data);
 }
 
+static int test_fps_helper(int color){
+	int fd_rtc = 0, fd_lcd = 0, p = 0;
```

Comment:
below is missing.

violations of coding rules
```
static int ...(int color)
{
    int fd_rtc = 0;
    int fd_lcd = 0;
    int p = 0;
```

Reply:
done

Reply:
Not done yet. Could you check the comment again?

Reply:
sir, I have already removed the test_fps_helper function. Since the test is now running like putarea(red) -> putarea(blue) -> putarea(red). So, this function was no longer required.